docs/queries/about.mdx

```diff
@@ -0,0 +1,418 @@
+    present. Throws an error if more than one match is found (use `queryAllBy`
+    instead if this is OK).
+  - `findBy...`: Returns a Promise which resolves when an element is found which
+    matches the given query. The promise is rejected if no element is found or
+    if more than one element is found after a default timeout of 1000ms. If you
+    need to find more than one element, use `findAllBy`.
+- Multiple Elements
+  - `getAllBy...`: Returns an array of all matching nodes for a query, and
+    throws an error if no elements match.
+  - `queryAllBy...`: Returns an array of all matching nodes for a query, and
+    return an empty array (`[]`) if no elements match.
+  - `findAllBy...`: Returns a promise which resolves to an array of elements
+    when any elements are found which match the given query. The promise is
+    rejected if no elements are found after a default timeout of `1000`ms.
+    - `findBy` methods are a combination of `getBy*` queries and
+      [`waitFor`](../dom-testing-library/api-async.mdx#waitfor). They accept the
+      `waitFor` options as the last argument (i.e.
+      `await screen.findByText('text', queryOptions, waitForOptions)`)
+
+<details>
+
+<summary>Summary Table</summary>
+
+<br />
+
+| Type of Query         | 0 Matches     | 1 Match        | >1 Matches   | Retry (Async/Await) |
+| --------------------- | ------------- | -------------- | ------------ | :-----------------: |
+| **Single Element**    |               |                |              |                     |
+| `getBy...`            | Throw error   | Return element | Throw error  |         No          |
+| `queryBy...`          | Return `null` | Return element | Throw error  |         No          |
+| `findBy...`           | Throw error   | Return element | Throw error  |         Yes         |
+| **Multiple Elements** |               |                |              |                     |
+| `getAllBy...`         | Throw         | Return array   | Return array |         No          |
+| `queryAllBy...`       | Return `[]`   | Return array   | Return array |         No          |
+| `findAllBy...`        | Throw         | Return array   | Return array |         Yes         |
+
+</details>
+
+## Priority
+
+Based on [the Guiding Principles](guiding-principles.mdx), your test should
+resemble how users interact with your code (component, page, etc.) as much as
+possible. With this in mind, we recommend this order of priority:
+
+1. **Queries Accessible to Everyone** queries that reflect the experience of
+   visual/mouse users as well as those that use assistive technology
+   1. `getByRole`: This can be used to query every element that is exposed in
+      the
+      [accessibility tree](https://developer.mozilla.org/en-US/docs/Glossary/AOM).
+      With the `name` option you can filter the returned elements by their
+      [accessible name](https://www.w3.org/TR/accname-1.1/). This should be your
```

Comment:
Did we link to this previously as well? I seem to remember we link to more high level explainer but that might've been on another page. 

At this point I don't find this particular link helpful.

Reply:
Didn't change that. Personally, I have used that resource and found it helpful.